### PR TITLE
skip configs download flag

### PR DESCRIPTION
**Pull request by miiu96**
- Added an extra flag that will not let the simulator download configs in case the flag is provided

### CODE REVIEW

File: cmd/chainsimulator/main.go [hunk 1]
```diff
@@ -334,6 +320,35 @@ func initializeLogger(ctx *cli.Context, cfg config.Config) (closing.Closer, erro
 	return fileLogging, nil
 }
 
+func fetchConfigs(skipDownload bool, cfg config.Config, nodeConfigs, proxyConfigs string) error {
+	buildInfo, ok := debug.ReadBuildInfo()
+	if !ok {
+		return errors.New("cannot read build info")
+	}
+	if skipDownload {
+		log.Warn(`flag "skip-configs-download" has been provided, if configs for node and proxy are missing simulator will not start`)
```
Comment (iulianpascalau) [inline cmd/chainsimulator/main.go:329]: if the configs*
are missing, then the simulator will not start *

Comment (miiu96) [reply cmd/chainsimulator/main.go]: done

File: cmd/chainsimulator/main.go [hunk 2]
```diff
@@ -334,6 +320,35 @@ func initializeLogger(ctx *cli.Context, cfg config.Config) (closing.Closer, erro
...
+	if err != nil {
+		return err
+	}
+
+	err = configsFetcher.FetchProxyConfigs(buildInfo, proxyConfigs)
+	if err != nil {
+		return err
+	}
+
+	return nil
```
Comment (ssd04) [inline cmd/chainsimulator/main.go:349]:
```suggestion
	return configsFetcher.FetchProxyConfigs(buildInfo, proxyConfigs)
```
?

Comment (miiu96) [reply cmd/chainsimulator/main.go:349]: fixed

**Review by ssd04 — DISMISSED** [review]

**Pull request merged**